code/datums/quirks/negative_quirks.dm

```diff
@@ -238,6 +238,41 @@
 	quirk_holder.clear_mood_event("family_heirloom_missing")
 	quirk_holder.clear_mood_event("family_heirloom")
 
+/datum/quirk/glassjaw
+	name = "Glass Jaw"
+	desc = "You have a very fragile jaw. Any sufficiently hard blow to your head might knock you out."
+	icon = "facegrimace"
+	value = -4
+	gain_text = "<span class='danger'>Your jaw feels loose.</span>"
+	lose_text = "<span class='notice'>Your jaw feels connected again.</span>"
```

Comment:
```suggestion
	gain_text = span_danger("Your jaw feels loose.")
	lose_text = span_notice("Your jaw feels connected again.")
```

Reply:
everything else uses manual spans but i'm sure that's not a good thing so sure.